src/vit-testing/vitup/committee_1

```diff
@@ -1 +1 @@
-ed25519e_sk1hpx0pkpd3qu7ww9nx87y6vswhrk2lvey3ac3xzl4va6hw5rhapfz6sy48tywfnhq9u47arzc0fah7rlrlsvh22jhmxxdv8rv38n0evg3fzf55
+ed25519e_sk1pzjv5ghwf4mzvrqus0u7trplspuvq78pkzuwmz7s9cer6kq3vffv5xsyd4crgmvsm2mrnl0fmujgt8eavwjjtwwrtcdez23kq4gfvjcrnjhjg
```

Comment:
I think we should discard all changes for 

src/vit-testing/vitup:
- block0.bin
- commitee_1
- genesis.yaml
- initial_setup.dot